coordinator/persistence-cassandra-3.11/pom.xml

```diff
@@ -215,7 +215,7 @@
               <goal>run</goal>
             </goals>
             <configuration>
-              <tasks>
+              <target>
```

Comment:
Actual fix: Maven-antrun plug-in's XML definition somehow changed in new(er) versions?